src/pages/MonthlyGood/KivaAsExpert.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+	<div class="kiva-as-expert-section-wrapper">
```

Comment:
I think this wrapper & background color should be moved to the monthly good page component, as it only matters in the context of that page.

Reply:
For optimization reasons, the image on this component has a background color and the component is going to look strange if the component background does not match the image background.

Reply:
👍